docker/dockerfiles/be/be_entrypoint.sh

```diff
@@ -109,18 +109,37 @@ collect_env_info
 add_self $svc_name || exit $?
 log_stderr "run start_be.sh"
 
+if [ $COREDUMP_ENABLED == "true" ]; then
```

Comment:
```suggestion
if [[ "$COREDUMP_ENABLED" == "true" ]]; then
```
avoid empty env var causes bash error.